models.py

```diff
@@ -25,10 +25,10 @@ class Meta:
         ordering = ['name']
 
     def books(self):
-        return Book.search(query='name:"%s"' % self.name )
+        return Book.search(query='genre_aat:books+AND+name:"%s"' % self.name )
 
     def prints(self):
-        return Print.search(query='contributor:"%s"' % self.name )
+        return Book.search(query='(genre_aat:"etchings (prints)"+OR+genre_aat:"engravings (prints)")+AND+name:"%s"' % self.name )
```

Comment:
Right now everywhere else in the project the prints and books are treated as the same type of object but sorted by the "genre_aat" field. The Print class isn't even used and the Book class has to be further narrowed by genre because it encompasses the prints as well. I agree that for good style it would be better to update the classes to make sure prints and books are treated as different things; for now I was just following how this had been "fixed" before to avoid a bug. Making the class distinction would involve more changes to how prints and books are distinguished throughout the project. I can make this structural change if you think it is necessary.